app.js

```diff
@@ -1,34 +1,37 @@
 require('dotenv').load();
 const express = require('express'),
-    app = express(),
-    bodyParser = require('body-parser'),
-    cors = require('cors'),
-    mongoose = require('mongoose'),
-    passport = require('passport'),
-    LocalStrategy = require('passport-local'),
-    methodOverride = require('method-override'),
-    // Product = require('./models/product'),
-    // Comment = require('./models/comment'),
-    User = require('./models/user');
+      app = express(),
+      bodyParser = require('body-parser'),
+      cors = require('cors'),
+      mongoose = require('mongoose'),
+      passport = require('passport'),
+      LocalStrategy = require('passport-local'),
+      methodOverride = require('method-override'),
+      Product = require('./models/product'),
+      Comment = require('./models/comment'),
+      User = require('./models/user');
 // seedDB          = require("./seeds")
 
 //Requiring Routes
-// const loginRoutes = require('./routes/login');
+const loginRoutes = require('./routes/login');
+const productRoutes = require('./routes/product');
+const commentRoutes = require('./routes/comments');
 
 app.get('/', (req, res) => {
-    res.send({
+    res.send({ 
         here: 'is',
         some: 'cool',
         data: '.'
     });
 });
+app.use("/comments", commentRoutes);
```

Comment:
`app.use("/comments", commentRoutes);` is both on line 27 and 55, let's delete one :)